index.html

```diff
@@ -58,10 +58,10 @@ <h1>RustFest {{site.location.city}} {{site.dates.year}}</h1>
 {% assign gold_sponsors = site.sponsors | where: "level", "gold" %}
 {% assign silver_sponsors = site.sponsors | where: "level", "silver" %}
 
-<!--
+
 <div class='popout sponsors'>
-  <section>
-    {% if platinum_sponsors[0] %}
+ <section>
+  <!--   {% if platinum_sponsors[0] %}
```

Comment:
This is a lot of commented lines. Shouldn't we just remove them?

Reply:
It was there previously, but I guess we can remove it for now